lua/neo-tree/defaults.lua

```diff
@@ -238,8 +238,8 @@ local config = {
       ["y"] = "copy_to_clipboard",
       ["x"] = "cut_to_clipboard",
       ["p"] = "paste_from_clipboard",
-      ["c"] = "copy", -- takes text input for destination
-      ["m"] = "move", -- takes text input for destination
+      ["c"] = "copy", -- takes text input for destination, also accepts the config.show_path option
+      ["m"] = "move", -- takes text input for destination, also accepts the config.show_path option
```

Comment:
Can you also add this comment to the README?
https://github.com/nvim-neo-tree/neo-tree.nvim/blob/6dac7b5b46c27af921369dc5ea1390025f0e58ae/README.md?plain=1#L212-L213